src/desktop/Workspace.cpp

```diff
@@ -442,6 +444,37 @@ bool CWorkspace::matchesStaticSelector(const std::string& selector_) {
                 continue;
             }
 
+            if (cur == 'f') {
+                if (!prop.starts_with("f[") || !prop.ends_with("]")) {
+                    Debug::log(LOG, "Invalid selector {}", selector);
+                    return false;
+                }
+
+                prop               = prop.substr(2, prop.length() - 3);
+                const auto FSSTATE = configStringToInt(prop);
```

Comment:
I'd rather use `std::stoi` here, as configStringToInt returns 0 on error

Just remember to wrap in a try catch

Reply:
fixed